server/webapp/WEB-INF/rails/webpack/models/auth_configs/spec/auth_configs_crud_spec.ts

```diff
@@ -75,7 +75,7 @@ describe("AuthorizationConfigurationCRUD", () => {
     expect(request.method).toEqual("DELETE");
     expect(request.data()).toEqual(toJSON({} as AuthConfigJSON));
     expect(request.requestHeaders.Accept).toEqual("application/vnd.go.cd.v1+json");
-    expect(request.requestHeaders["Content-Type"]).toEqual("application/json; charset=utf-8");
+    expect(request.requestHeaders["Content-Type"]).toEqual(undefined!);
```

Comment:
why this change?

Reply:
As per the [mithril's changelog](https://github.com/MithrilJS/mithril.js/releases) it is stated that the request header `https://github.com/MithrilJS/mithril.js/releases` is set for all XHR methods by default, **provided they have a body `that's != null`**.

In case of delete, we don't send any request body, hence we are not required to set this header.
This assertion is just to verify, we are not setting this header (outside of mithril) for delete requests.